products/pan-gp.md

```diff
@@ -36,14 +36,14 @@ releases:
 
 -   releaseCycle: "5.2"
     eol: 2023-02-28
-    support: 2023-02-28
+    support: 2024-02-28
```

Comment:
2024 is the EOL Date, support date is  2023 31 August.
```suggestion
    support: 2023-08-31
```

Reply:
correct, I swapped eol and support :(